src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="forms">
+      <div class="form-big">
+        <form
+          action="#"
+          method="get"
+          data-qa="big"
+          class="searchbar"
+        >
+          <img
+            src="images\Search.svg"
+            alt="icon"
+            class="icon"
+          >
+          <input
+            type="text"
+            inputmode="search"
+            name="bigSearchBar"
+            data-qa="keypress"
+            placeholder="Try &quot;Los Angeles&quot;"
+            class="input"
+          >
+        </form>
+      </div>
+      <div class="form-small">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      </div>
      
      <div class="form-small">
```